tests/test_cases/test_vhdl_zerovector/test_vhdl_zerovector.py

```diff
@@ -0,0 +1,49 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import cocotb
+from cocotb.result import TestFailure
+from cocotb.triggers import Timer
+
+
+@cocotb.test()
+async def test_long_signal(dut):
+    """ Write and read a normal signal (longer than 0)."""
+    await Timer(1, "ns")
```

Comment:
```suggestion
```
Not necessary. The particular scheduler bug that required you to do this has been fixed.